### ISSUE TITLE
Cutlets can't be grilled

### ISSUE THREAD

**Issue by irismessage** [space-wizards/space-station-14#29740]
## Description
Meat can be grilled to make cooked meat, plus both raw and cooked meat can be sliced to get raw and cooked cutlets. However raw cutlets can't be grilled once sliced

```
          Grill
Raw meat ------>  Cooked meat
|                  |
| Slice            | Slice
v                  v
Raw cutlet --xx--> Cooked cutlet
          Missing link
```

**Reproduction**
Slice meat into raw cutlets then place it on a grill. wont cook

**Comment by joelsgp** [comment]
Also I'm curious how to make fried chicken out of raw chicken? Is that currently possible?

**Comment by deltanedas** [comment]
gordon ramsey would be seething

**Comment by lzk228** [comment]
gonna look for all kind of meat and make it cookable🔥

**Comment by themias** [comment]
Fixed by #31048
